monitoring/README.md

```diff
@@ -1,19 +1,17 @@
-# Audius Network Monitoring
-### Launch Grafana
+### Launch Prometheus & Grafana
 
 ```bash
-docker-compose up -d grafana prometheus
+docker-compose up -d prometheus grafana
 ./grafana/bin/create-data-sources.sh
 ./grafana/bin/upload-dashboards.sh
 ```
 
-Visit `http://${REMOTE_DEV_HOST}:3000`.
+Prometheus is accessible at port 9090, and Grafana at port 3000
 
 ### Deploy Changes
 
+Inside `monitoring-tools repo`
```

Comment:
I'll modify these two changes in my branch to differentiate between local and production deployments.